test/TEST-46-HOMED/test.sh

```diff
@@ -3,9 +3,21 @@
 set -e
 
 TEST_DESCRIPTION="testing homed"
-TEST_NO_QEMU=1
+
+# Skip the qemu version of the test, unless we have btrfs
+(modprobe -nv btrfs && command -v mkfs.btrfs) || TEST_NO_QEMU=1
```

Comment:
loading the module might fail on the CI builder, no? eg, if it's running in a container, but it doesn't mean it would fail in qemu

Reply:
oh wait it's a dry-run, never mind